src/shared/types.ts

```diff
@@ -0,0 +1,5 @@
+import { LinkList } from '@/widgets/required/required.types';
+
+export type ListData = (string | LinkList)[] | [] | undefined;
```

Comment:
Do we really ned `undefined` here?

Reply:
In some data-files (e.g - `dev-data/js-ru-path.data.ts`) we don't have the `list` parameter. I suppose we'll be able to remove `undefined` after data-files refactoring

Reply:
Yeah, but we don't have the `list` prop there because it's an optional property
![image](https://github.com/user-attachments/assets/542f1b0b-3124-4385-a704-29f48cd30436)

So, i'd say the type should look like this: `export type ListData = (string | LinkList)[];`